### ISSUE TITLE
Mision[Dragona, desencadenada]

### ISSUE THREAD

**Issue by misteriosoccmc** [RoDNet/Bugtracker-Infinium-4.0.6a#798]
Al parecer esta mision http://es.wowhead.com/quest=27701/dragona-desencadenada, tira el server, la intente hacer 2 veces y las 2 veses lo hizo

**Comment by ghost** [comment]
Pues que raro, creo que la semana pasada o ante pasada la hice y no se cayó el sv.

**Comment by redneo** [comment]
Efectivamente, el spell 86466 hace que se caiga el server. Ya he borrado el gob que hace que se pueda lanzar ese spell hasta que se descubra el porque del crash.

**Comment by ghost** [comment]
Pero la quest se puede completar sin usar el spell,solo le das clic derecho a la dragona y se completa.
